### PR TITLE
Changed CLI to use --types flag, a comma separated list of desired an…

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

**Comment by aaron-prindle** [comment]
tests and README updated.  updating `container-diff help` or other messaging to include available analyzers can be done in another PR

**Comment by dlorenc** [comment]
Looks like we accidentally added a "main" file to the repo that's now conflicting. You'll have to delete.

**Comment by aaron-prindle** [comment]
deleted 'main'

File: cmd/diff_test.go [hunk 1]
```diff
@@ -13,7 +13,8 @@ var diffArgNumTests = []testpair{
 
 func TestDiffArgNum(t *testing.T) {
 	for _, test := range diffArgNumTests {
-		valid, err := checkDiffArgNum(test.input)
+		err := checkDiffArgNum(test.input)
```
Comment (dlorenc) [inline cmd/diff_test.go]: This is a bit weird of a construct. Do we need err and valid? Could you do something like:

valid := (checkDiffArgNum == nil) ?

Comment (aaron-prindle) [reply cmd/diff_test.go]: good point, changed

**Comment by aaron-prindle** [comment]
the tests output the err on unexpected errors so storing it separately is needed.  changing it back

**Comment by aaron-prindle** [comment]
i can just use (err == nil) in place of valid

**Pull request merged**